server/modules/authentication/discord/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "wikijs-discord-oauth2",
+  "version": "1.0.0",
+  "description": "Supports using Discord as a SSO source, and allows the administrator to configure discord groups that are required for authentication.",
+  "main": "authentication.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "wikijs",
+    "discord",
+    "sso"
+  ],
+  "author": "Tom Dakan",
+  "license": "AGPL-3.0-or-later",
+  "dependencies": {
+    "discord-oauth2": "^2.11.0"
+  }
+}
```

Comment:
Is this package.json needed? I assume you've added those changes accidentally, as you said you're binding your bundle on your own instance, but I would remove this file also with `package-lock.json` 😁

Reply:
@tboba I added the package to the generic implementation of the discord authentication module with the intention that my changes could extend the generic discord authentication module. I don't think my change warrants a new module since the group-based authentication is fully optional. 

So I think it should be there based on my read of the documentation, but let me know if I'm misunderstanding something.

Reply:
@TomDakan I still don't think that you should create your own `package.json` here - it's inconsistent with other authentication methods (take a look at the hierarchy: every component should have just `authentication.js` and `definition.yml` file. Also, there's only one generic `package.json` file in the project which serves for the whole project 🤔

Reply:
I'm doing this based on the documentation here: https://docs.requarks.io/dev/modules where it says

"Some modules may require npm dependencies not present in the main Wiki.js package.json. Verify first that the dependency you need isn't already in the main project. DO NOT add dependencies to the main package.json. This is reserved for the core Wiki.js app only and doing so can introduce breaking changes.

If the package you need isn't already present, you must instead install extra dependencies within the same folder as your module definition file. You must first create a local package.json from your module directory:"

Reply:
Okay, sure! Thanks for justifying that.